docs/fft.rst

```diff
@@ -0,0 +1,170 @@
+###################################################
```

Comment:
Should this be linked from the index.rst? BTW, thanks for making this! Maybe add to the set in the 'library examples' section?

Reply:
Good point. I normally just use the sidebar (and assume it will be added there automatically), but good to add this here too!